spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'User post index page', type: :feature do
```

Comment:
- [ ] Kindly make sure to include a test that redirects to the post's show page when you click on a post.